### PR TITLE
Auth forms are now not dialogs

### PR DESCRIPTION
**Pull request by ihor-il**
### Set up routing:
- Wildcard: redirect to `auth` 
- `auth`: shared component with background and logo, redirecting to `auth/login`
- `auth/login`: login form
login form
- `auth/register`: register form
register form

------
@DaniilPoiarkov and @oleksandr-vovk, please check if all styles and interactions are the same as in your respective PRs (#34, #42)

### CODE REVIEW

File: frontend/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (Its-OP) [inline frontend/.vscode/launch.json]: Is this file supposed to be here ?

**Review by Its-OP — CHANGES_REQUESTED** [review]
I think we should have an "Identity" module, that would include 3 components - auth, login and register pages. If you decide to go modular way, please, add identity-routing module for these pages as well.

**P.S.** Everything above is just a suggestion. I've requested changes because I believe that `.vscode` folder was added to this PR mistakenly.

File: frontend/.vscode/launch.json (hunk 1 above)
Comment (ihor-il) [reply frontend/.vscode/launch.json]: idk, gitignore states that it does but i can remove it

Comment (ihor-il) [reply frontend/.vscode/launch.json]: I totally agree about modular structure, but this pr is only for remaking forms as pages and not dialogs. I will move those files into separate module after adding `Restore password` page and setting up auth flow

File: frontend/src/app/components/login-page/login-page.component.html [hunk 2]
```diff
@@ -1,56 +1,51 @@
...
+          placeholder="Enter your email"
+          formControlName="emailControl"
+        />
+        <span class="error" *ngIf="emailControl.errors?.['required'] && (emailControl.dirty || emailControl.touched)">Email is required.</span>
+        <span class="error" *ngIf="emailControl.errors?.['email']">Incorrect email format</span>
+      </div>
+      <div class="input-container">
+        <div class="label-container full-width">
+          <label>Password</label>
+          <a class="forget-pass" href="/">Forgot password?</a>
```
Comment (oleksandr-vovk) [inline frontend/src/app/components/login-page/login-page.component.html:22]: Add a link to "Forgot password" page.

**Review by oleksandr-vovk — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/login-page/login-page.component.html (hunk 2 above)
Comment (ihor-il) [reply frontend/src/app/components/login-page/login-page.component.html:22]: we don't have it yet

File: frontend/src/app/components/auth-page/auth-page.component.sass [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  align-items: center
+  margin: -10px -10px -10px -10px
+
+.logo-container
+  height: 100%
+  background-position: center
+  background-repeat: no-repeat
+  background-size: cover
+  background-image: url('/assets/background.png')
+  cursor: pointer
```
Comment (oleksandr-vovk) [inline frontend/src/app/components/auth-page/auth-page.component.sass]: Remove "cursor: pointer" rule, it is not useful now

**Review by oleksandr-vovk — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/auth-page/auth-page.component.sass (hunk 3 above)
Comment (ihor-il) [reply frontend/src/app/components/auth-page/auth-page.component.sass]: done

**Review by oleksandr-vovk — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by Its-OP — APPROVED** [review]

**Pull request merged**